wallet/mixing.go

```diff
@@ -479,23 +478,30 @@ func (w *Wallet) MixAccount(ctx context.Context, changeAccount, mixAccount,
 	w.lockedOutpointMu.Unlock()
 
 	var g errgroup.Group
+	var success int
```

Comment:
Shouldn't this be an `atomic.Int`?  It's being modified by goroutines concurrently and it isn't protected by a write barrier of any type at the moment.

Reply:
yes, thx